packages/arb-token-bridge-ui/src/components/syncers/TokenListSyncer.tsx

```diff
@@ -22,7 +24,11 @@ const TokenListSyncer = (): JSX.Element => {
       return
     }
 
-    if (!arbTokenBridge?.walletAddress) {
+    if (!arbTokenBridgeLoaded) {
+      return
+    }
+
+    if (typeof walletAddress === 'undefined') {
```

Comment:
nit: for the sake of consistency should we just go with `!walletAddress` everywhere?